### PR TITLE
Add respondent home ui

### PR DESCRIPTION
**Pull request by liamtoozer**
# Motivation and Context

This adds the respondent-home-ui application. The image in DockerHub is currently based on the Dockerfile from this branch: https://github.com/ONSdigital/respondent-home-ui/blob/ssd/Dockerfile.

# What has changed

* `.env` - Had to base64 encode multiline keys to workaround newlines with docker compose reading env vars from file, see [here](https://goo.gl/GXqLCv). I could have used the multiline YAML inline in `rm-services.yml` but I tried to keep everything consistent with the other apps. 
* `rm-services.yml` - Added new app and env vars.

# How to test?

* Run the following:
```bash
docker-compose -f rm-services.yml up -d respondent-home-ui
```
* Open a browser and navigate to [localhost:9292](localhost:9292)
* Check that the UI loads correctly.

# Links

[Trello](https://trello.com/c/SwkUMf9O/99-dev-task-7-ext-access-eq-social-survey-rh-8)

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -133,3 +133,15 @@ COMMS_TEMPLATE_PORT=8182
...
+
+# Respondent Home UI
+RESPONDENT_HOME_UI_PORT=9292
+RESPONDENT_HOME_ANALYTICS_ACCOUNT=UA-56892037-7
+RESPONDENT_HOME_EQ_HOST=eq-server
+RESPONDENT_HOME_EQ_PORT=5000
+RESPONDENT_HOME_EQ_PROTOCOL=http
+RESPONDENT_HOME_IAC_SERVICE_PROTOCOL=http
+RESPONDENT_HOME_LOCALE=en
+# base64 encoded keys to workaround newlines issue with .env files (https://goo.gl/GXqLCv)
```
Comment (benjefferies) [inline .env]: Nice use of a short link 👍

File: .env [hunk 2]
```diff
@@ -133,3 +133,15 @@ COMMS_TEMPLATE_PORT=8182
...
...UXlyM01LRG5rNVRPVHk2b1VVWHBKcTBFWkRvcWd1cVJmZwo5aU1sWnlqL1FxemNvdDdBaXh3dUJORmdBTGlFWHNwZDNROXVTTHh1d0xLc1dlT3JJc3ZVR1pRd2xkSXRTSUQzCjd1d2I4UXIzVE9BWm1PUGlwYVFsNzFxY0E0STA4bHErU3A5S2ZkMEVORVcyV0FMVFRDMzBmUVVDZ1lFQXcrUm8KSEFkNnJ4Q0l5REloNm5uazY5UFM5K2l1UjZILzZCYzB2VTFhYWdzekRRa3RwazBVRGtqYWFNUTZSSGVialpqVwpzekJyc0Q1RUZUNUdMeExqMHZiVUp2V3pLNis2bHFPemlzZGlJdTBVMTFCM2VMNk5FdmFBZHNnU3E5UWo0WDZOCnJZaDV4MFc1RTJWSW9EQWUxTmlnbEphZ2lhSFV2RjgyUzE1WEw1c0NnWUVBa3hJMFA0b3krTWQybTBueVZMb04KV3Z4b0ZRNU1PVnNLeGs1ZERhMkJCQW5HRWk0UlNnTm9TQTdiV2lXelgra1dwT05ZK1l5d0ozWXdid2pMMzBOeAoyL1Z4YmRSdnA2YzhXV3FkVzZVeld3TGowT3RmMlF6eTJFVEVMdUFYanFpRjZNK0FiM2VqZk5pNVBBem1JSHpDCkhXRCtuaWNCeGg0dktTL2lCMkVDQ3NrQ2dZQUx0SDQwRG81YUg1Yk12NUlmUEtGZEpnOFpHU0l3elpGSkFQbjUKeGNGOVRLWUVSV1FnNThJbkNGdGtuelVNR2ttRk1IakR4RVRyQncxeVo0SXlGZUxmSERpWkMxMWxZdDYzV1pqRQoxaDBiRVNlSnBoazljY1ZrT0s1OGFvQVRIRjM0ZjJOZjF5UFBzRzdhcGRhdEc3TDc3T1hMZXlTa2xMTHRmcEx6CjlmTW51UUtCZ0ZKQ2xoMnZOWUZsTm5kWG9WNU5GUExOejFMNVd2NVdOd1djdUdPNGlDc1VoMmpGVmw0M3o0NTkKVjdadVJwLy80dkl6RU9OcXF5TFVGdDFKWkt1Q3dXUUIyc1JqS1dwcVJCMDdDMG8yNDNYZWZENWY3ODQ0bm1YcQpIMHhLTUZSWWpEcWhGZFRYRldWUG5rL3R1L014b1NUZEswRmZzZ1VuanMrVEd2TjVUbkJXCi0tLS0tRU5EIFJTQSBQUklWQVRFIEtFWS0tLS0tCgo=
```
Comment (madeye-matt) [inline .env]: Is this the private key unencrypted?

Comment (liamtoozer) [reply .env]: Yeah, after it has been base64-decoded it gives the private key data similar to this:
```
-----BEGIN RSA PRIVATE KEY-----
...key...
-----END RSA PRIVATE KEY-----
```
I used keys from this repo (https://github.com/onsdigital/ras-frontstage/blob/fee057c93a17388d8f042a3833c33daf2742cfa7/tests/test_data/jwt-test-keys/test_key.json#L1)

**Comment by liamtoozer** [comment]
Closing for now in favour of moving the application from Ruby to Python. Will raise another PR when that's ready to go.

**Pull request closed without merging**